<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GlassTypeManageDialog</class>
 <widget class="QWidget" name="GlassTypeManageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>玻璃型号管理</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>391</width>
     <height>481</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addBtn">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>20</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>添加玻璃型号</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editBtn">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>60</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>修改型号配置</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
